deployment/ansible/roles/nyc-trees.app/templates/nginx-nyc-trees-app.conf.j2

```diff
@@ -6,6 +6,20 @@ server {
 
   access_log /var/log/nginx/nyc-trees-app.access.log logstash_json;
 
+  location /favicon.ico {
```

Comment:
I have to re-test today. After provisioning in test mode, the favicon file at that path didn't have a hash.